### DISCUSSION TITLE
MeetingSummarizer - Effortlessly record, transcribe, and summarize meetings

### DISCUSSION THREAD

**Discussion by rajpdus** [openai/whisper#1235] [Show and tell]
MeetingSummarizer is a Python desktop utility that allows users to record meetings and automatically generate a summary of the conversation. The utility uses the ffmpeg library to record the meeting, the OpenAI Whisper module to transcribe the recording, and the OpenAI GPT-3.5-Turbo model to generate a summary of the conversation.

Users can choose to use the command-line interface or the graphical user interface to interact with the utility. The GUI provides an easy-to-use interface for starting and stopping recordings and viewing the transcription and summary of the meeting. The CLI interface provides more advanced options for users who want to customize the recording and transcription process.

The project is open source and available on GitHub at [MeetingSummrizer](https://github.com/rajpdus/MeetingSummarizer) The code is written in Python 3.10 and uses several third-party libraries, including ffmpeg, PyQt5.

**Comment by vinivyes** [comment]
There is a typo on the link taking to a 404 page @rajpdus it is adding a dot (.) at the end

**Reply by rajpdus** [reply]
Thanks @vinivyes. Fixed the link.
